lib/modules/manager/ocb/extract.ts

```diff
@@ -0,0 +1,78 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { regEx } from '../../../util/regex';
+import { parseSingleYaml } from '../../../util/yaml';
+import { GoDatasource } from '../../datasource/go';
+import type {
+  ExtractConfig,
+  PackageDependency,
+  PackageFileContent,
+} from '../types';
+import { type Module, type OCBConfig, OCBConfigSchema } from './schema';
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string,
+  _config?: ExtractConfig,
+): PackageFileContent | null {
+  let definition: OCBConfig | null = null;
+  try {
+    const yaml = parseSingleYaml(content);
+    const parsed = OCBConfigSchema.safeParse(yaml);
+    if (!parsed.success) {
+      logger.trace(
+        { packageFile, error: parsed.error },
+        'Failed to parse OCB schema',
+      );
+      return null;
+    }
+
+    definition = parsed.data;
+  } catch (error) {
+    logger.trace({ packageFile, error }, 'Failed to parse OCB file as YAML');
```

Comment:
```suggestion
    logger.debug({ packageFile, error }, 'Failed to parse OCB file as YAML');
```
I think this should be visible on debug log